airflow/providers/databricks/CHANGELOG.rst

```diff
@@ -61,6 +61,13 @@ Misc
 4.5.0
 .....
 
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+The ``offset`` parameter has been deprecated from ``list_jobs`` in favor of faster pagination with ``page_token`` similarly to `Databricks API <https://docs.databricks.com/api/workspace/jobs/list>`_.
```

Comment:
It's not in the right place it should be at the top of the log before the
```
Changelog
---------
```

Reply:
Gotcha, fixed!